src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -218,11 +197,11 @@ MetaView::unchecked_insert(
         using namespace std;
         items_.emplace_hint(iter, piecewise_construct,
             forward_as_tuple(sle->key()),
-                forward_as_tuple(move(sle),
-                    taaCREATE, mSeq));
+                forward_as_tuple(taaCREATE,
+                    move(sle)));
```

Comment:
I was worried about `foo(sle->key(), std::move(sle));` where `foo` invokes a move constructor for the second parameter. The call `sle->key()` might be `nullptr->key()`.